docs/devverification-guide.md

```diff
@@ -0,0 +1,55 @@
+#Dev Guide to Verify Each Steps
+
+This document can help you to verify each step is done correctly while setting up a development environment.
+It contains information about some of the steps but not all, which you can refer after completing eact step mentioned [here](https://github.com/TEAMMATES/teammates/blob/master/docs/settingUp.md).
+
+The steps given for setting up a developer environment works with Linux or Mac or Windows only with these pointers :
+- Replace `./gradlew` to `gradlew.bat` if you are using Windows.
+- All the commands are assumed to be run from the root project folder, unless otherwise specified.
+- When a version is specified for any tool, install that version instead of the latest version available.
+
+## Step 1: Install necessary tools and languages
+
+After installing JDK 7 check the version of Java and the output will be something like this :
+
+`java version "1.7.0_67"`  
+
+## Step 2: Obtain your own repository copy
+
+Check if the upstream is set properly by using this command `git remote -v` and the output will be something like this :
+
+`upstream        https://github.com/TEAMMATES/teammates.git (fetch)`
+
+`upstream        https://github.com/TEAMMATES/teammates.git (push)`
+
+More information can be found at this [documentation](https://help.github.com/articles/fork-a-repo/).
+
+## Step 3: Set up project-specific settings and dependencies
+
+After the execution of the respective commands in each step the following folders will be created.
+
+1. `./gradlew appengineDownloadSdk` will create the folders `.gradle` and `.git`  in the project root directory.
```

Comment:
This is inaccurate. `.git` is created the moment you cloned the repository, and `.gradle` is created the moment you run any Gradle wrapper script on the root project folder.

The only correct method to verify this is checking the user's Gradle folder for the existence of the SDK.